spec/controllers/users/registrations_controller_spec.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
+
+RSpec.describe Users::RegistrationsController do
+  let(:user) { create(:user) }
+
+  before do
+    request.env['devise.mapping'] = Devise.mappings[:user]
+    sign_in user
+  end
+
+  describe '#edit' do
+    before do
+      get :edit
+    end
+
+    it 'renders the edit template' do
+      expect(response).to render_template(:edit)
+    end
+
+    it 'returns status 200' do
+      expect(response).to have_http_status(:ok)
+    end
+  end
+
+  describe '#action' do
```

Comment:
Do you have action which is calls `action`?
As I see, from the context of the test, you are describe the `put` action
So call this group of tests as expected
```suggestion
  describe '#put' do
```